aim/sdk/artifacts/text.py

```diff
@@ -0,0 +1,32 @@
+from abc import ABCMeta
+import time
+
+from aim.sdk.artifacts import Record
+from aim.sdk.artifacts.artifact import Artifact
+
+
+class Text(Artifact, metaclass=ABCMeta):
+    cat = ('text',)
+    _step_counter = {}
+
+    def __init__(self, document_name: str, text: str):
+        self.document_name = document_name
+        self.text = text
+        self.timestamp = int(time.time())
+        self.json = {"text": self.text, "timestamp": self.timestamp}
```

Comment:
Maybe we should rename `json` property to smth more specific, e.g. `content`? And move this line to`serialize` method, cause constructing actual content is a part of "serilization", right?